search_assistant/src/search_assistant.py

```diff
@@ -32,15 +31,14 @@
 
 from utils.model_wrappers.api_gateway import APIGateway
 from utils.vectordb.vector_db import VectorDb
-from utils.visual.env_utils import get_wandb_key
 
 CONFIG_PATH = os.path.join(kit_dir, 'config.yaml')
 PERSIST_DIRECTORY = os.path.join(kit_dir, 'data/my-vector-db')
 
 load_dotenv(os.path.join(repo_dir, '.env'))
 
 # Handle the WANDB_API_KEY resolution before importing weave
-wandb_api_key = get_wandb_key()
+wandb_api_key = None  # get_wandb_key()
```

Comment:
why is this being initialized with None now?

Reply:
Fixed!